src/CustomerioTracking.tsx

```diff
@@ -31,6 +38,7 @@ const CustomerioReactnative = NativeModules.CustomerioReactnative
     );
 
 class CustomerIO {
+  static segmentClient: SegmentClient; // global variable declaration
```

Comment:
Is there a reason why this is `static`? 

If there is, I would appreciate a comment added explaining why for future reference.

Reply:
`static` properties can be used without creating an instance of the class. All other methods of this class are `static` too.